schema/ridl/_example/example1.ridl

```diff
@@ -39,15 +47,23 @@ message User
     + go.tag.db = created_at
     
 
-message Notice
+struct Notice
   - msg: string
 
 
-service ExampleService
+
+# syntax is: error <code> <name> <message> [-- <http status code>]
```

Comment:
new error keyword :) of course check the webrpc.json files too to see how it looks in that structure